Code/Tools/ProjectManager/Source/Application.cpp

```diff
@@ -24,6 +27,73 @@
 #include <QInputDialog>
 #include <QIcon>
 
+#if defined(AZ_PLATFORM_WINDOWS)
+LONG WINAPI CreateMiniDump(struct _EXCEPTION_POINTERS* exceptionPointers)
+{
+    AZ::IO::FixedMaxPath dumpPath{ "@log@/O3DEDump.dmp" };
+    if (auto fileIoBase = AZ::IO::FileIOBase::GetInstance(); fileIoBase != nullptr)
+    {
+        dumpPath = fileIoBase->ResolvePath(dumpPath, "@log@/O3DEDump.dmp");
+    }
```

Comment:
```suggestion
    AZ::IO::FixedMaxPath dumpPath{ "o3de.dmp" };
```

I think it's best to be paranoid and assume all AZ code like `FileIOBase` might be unsafe/corrupt and just write the .dmp directly to the current working directory or to %TEMP%.  Current directory might fail if the application is in a protected location like `Program Files`.  

Does our current game minidump handling do this?

Reply:
Improved the path generation to clean it up better. Also use a backup path adjacent to the working directory.